src/app/system/manage-scheduler-jobs/view-scheduler-job/view-scheduler-job.component.html

```diff
@@ -0,0 +1,57 @@
+<div fxLayout="row" fxLayoutAlign="end" fxLayoutGap="2%" fxLayout.lt-md="column" class="container m-b-20">
```

Comment:
make the structure of this html consistent with view tax component.html, take care of indentations.